src/views/virtualmachines/tree/components/TreeViewRightClickActionMenu/DefaultRightClickActionMenu.tsx

```diff
@@ -20,7 +20,7 @@ const DefaultRightClickActionMenu: FC<DefaultRightClickActionMenuProps> = ({
   const { namespace, prefix } = getElementComponentsFromID(triggerElement);
 
   const vms = getVMsTrigger(triggerElement);
-  const actions = useMultipleVirtualMachineActions(vms);
+  const actions = useMultipleVirtualMachineActions(vms, vms);
```

Comment:
If u are sending the same object, make the second one optional in the hook; if it doesn't exist, treat the first as the second